tools/yapf.sh

```diff
@@ -24,7 +24,7 @@ set -e
 function follow_links() (
   cd -P "$(dirname -- "$1")"
   file="$PWD/$(basename -- "$1")"
-  while [[ -h "$file" ]]; do
+  while [[ -L "$file" ]]; do
```

Comment:
Curious about the reason for this change. Unfortunately, there's a copy of `follow_links()` in nearly every shell script, so if this is fixing an issue, it likely exists in several other places.

Reply:
I think this was a formatter change (wow), and I agree in isolation it's confusing. I'll revert it before merging.